lib/cms-api.ts

```diff
@@ -28,7 +29,9 @@ let cmsApi: {
   getAllJobs: () => Promise<Job[]>;
 };
 
-if (process.env.DATOCMS_READ_ONLY_API_TOKEN) {
+if (process.env.STRAPI_API_URL) {
```

Comment:
This should be moved to the end, instead of first 👍

Reply:
Totally!! I forgot to update that, thanks 😉